deps/build.jl

```diff
@@ -33,21 +32,34 @@ end
 libfftw = library_dependency(libfftw_name, aliases=makealiases(libfftw_name))
 libfftwf = library_dependency(libfftwf_name, aliases=makealiases(libfftwf_name))
 
-provides(AptGet, "libfftw3-double3", [libfftw], os=:Linux)
-provides(AptGet, "libfftw3-single3", [libfftwf], os=:Linux)
-provides(Pacman, "fftw", [libfftw, libfftwf], os=:Linux)
-provides(Zypper, "libfftw3_threads3", [libfftw, libfftwf], os=:Linux)
-provides(Yum, "fftw-libs", [libfftw, libfftwf], os=:Linux)
-provides(BSDPkg, "fftw3", [libfftw, libfftwf], os=:FreeBSD)
+url = "https://github.com/ararslan/fftw-builder/releases/download/v\$FFTW_VER/libfftw-\$FFTW_VER"
```

Comment:
Yeah, this huge `include_string()` is nasty; we should instead just use `include()` and put the newer build instructions in a separate file